src/TigerPOMDPs.jl

```diff
@@ -81,7 +77,7 @@ end
 
 discount(pomdp::TigerPOMDP) = pomdp.discount_factor
 
-function generate_o(p::TigerPOMDP, s::Bool, rng::AbstractRNG)
-    d = observation(p, 0, s) # obs distrubtion not action dependant
+function gen(::DDNOut{:o}, p::TigerPOMDP, s::Bool, rng::AbstractRNG)
```

Comment:
DDNNode - we probably need to document/do something better for initial observations.